docs/source/releasenotes.rst

```diff
@@ -5,6 +5,7 @@ Release notes
 `Upcoming release <https://github.com/robocorp/rpaframework/projects/3#column-16713994>`_
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+- Library **RPA.FileSystem** (:pr:`568`): Add keyword examples and type hints.
```

Comment:
This shouldn't exist in this PR, I will remove it but not push until a full review.